src/telliot_ampl_feeds/feeds/usd_vwap.py

```diff
@@ -0,0 +1,11 @@
+"""Example datafeed used by AMPLUSDVWAPReporter."""
+from telliot.datafeed import DataFeed
+from telliot.queries.legacy_query import LegacyRequest
+from telliot_ampl.config import AMPLConfig
```

Comment:
`telliot_ampl` is an unresolved name here.  
Do you mean `telliot_ampl_feeds` ?

Reply:
Thanks for spotting